Bluewire.Conventions/BranchSemantics.cs

```diff
@@ -30,5 +31,25 @@ public string[] GetRemoteBranchFilters(params BranchType[] types)
         {
             return types.Distinct().Select(t => t.BranchFilter).Where(b => !String.IsNullOrWhiteSpace(b)).Select(b => $"*/{b}").ToArray();
         }
+
+
+        // Assumes we always create a tag when we start a new version
+        public Ref GetVersionZeroRef(SemanticVersion semVer)
+        {
+            return new Ref(semVer.Major + "." + semVer.Minor);
+        }
+
+        // Assumes sementics defined in BranchType.cs
+        public Ref GetVersionLatestRef(SemanticVersion semVer)
+        {
+            switch (semVer.SemanticTag)
+            {
+                case "beta": return new Ref("master");

```

Comment:
I think it's correct as is and is working as expected but I suspect you are referring to what happens if the user submits an invalid combination of sprint and build number to the tool? I agree we could enhance the caller in future to notify the user of their mistake but I'm not convinced it's worthwhile straight away since such an edge case falls outside of the needs of release notes generation improvement. Happy to discuss priorities for further enhancements next week though.